packages/react/package.json

```diff
@@ -13,8 +13,8 @@
     "react"
   ],
   "sideEffects": false,
-  "main": "./lib/index.es.js",
-  "types": "./lib/index.d.ts",
+  "main": "./src/index.ts",
```

Comment:
This change will make the library can't be used without a bundler. Why we need this change?

Reply:
Apologies, the modifications mentioned above were made during my local testing and I forgot to revert them before committing. I will make sure to refine them along with the other sections.